buildrelease.sh

```diff
@@ -1,27 +1,28 @@
 #!/bin/bash
 
 # Script to perform a release build for GAX, based on
-# a tag which should already have been created.
-# This builds both GAX and the common protos; you should only push
-# the nuget packages you actually want to update.
-# Sample: buildrelease.sh Google.Api.Gax-2.1.0
+# a commit for which tags have been created.
+# This builds both GAX and the common protos; but will delete
+# all built packages that don't match one of the tags in the commit
+# hash. For instance, if there's only one tag in the commit hash 
+# that starts with Google.Api.Gax.Grpc then only the following nuget
+# packages will remain after the script is done:
+# Google.Api.Gax.Grpc.<version>.nupkg
+# Google.Api.Gax.Grpc.Gcp.<version>.nupkg
+# Google.Api.Gax.Grpc.Testing.<version>.nupkg
+# Sample: buildrelease.sh 847118821d8a8b4ac2ea3bbe692acea29c14c4da
```

Comment:
So to check, if we tag "Google.Api.Gax" (which is what we normally do) that will push everything apart from CommonProtos, right?

Reply:
Exactly, and if you tag Google.Api.Gax.Grpc that will push only the Google.Api.Gax.Grpc.* packages.
And you can have two tags on the same commit, one for Google.Api.Gax.Grcp and one for Google.Api.CommonProtos which will push Google.Api.Gax.Grcp.* and CommomProtos but not the other Google.Api.Gax packages.
I checked and noticed that the tags are "always" like this. I added examples on the PR comments.